components/common/Buttons.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { ButtonProps, Pressable, StyleSheet } from "react-native"
+import { Headline } from "@components/Text"
+
+const darkColor = "#26282A"
+const hexAlpha = "26" // 15% Opacity
+
+export const DarkButton = (props: ButtonProps) => {
```

Comment:
I think `Button` and `OutlinedButton` are better names, but we may need a better name for the former since it clashes with react-native's buttons.